docs/wiki/ALZ-Policies-Extra.md

```diff
@@ -17,6 +17,7 @@ ALZ provides several additional policies that are not assigned by default but th
 | Deny-Vnet-Peering | Deny vNet peering | Use to prevent vNet peering under specific scopes (e.g., Sandbox management group) |
 | Deny-Vnet-Peering-To-Non-Approved-Vnets | Deny vNet peering to non-approved vNets | Use to control vNet peering under specific scopes, like in the Corp management group, only allow peering to the hub vNet. |
 | Deploy-Budget | Deploy a default budget on all subscriptions under the assigned scope | Set a default budget for a specific scope, like setting a $500 budget on all subscriptions in the Sandbox management group |
+|Deploy-Sql-Security_20240529| Deploy-SQL Database built-in SQL security configuration| Deploy auditing, Alert, TDE and SQL vulnerability to SQL Databases when it not exist in the deployment|
```

Comment:
```suggestion
|Deploy-Sql-Security_20240529| Deploy-SQL Database built-in SQL security configuration| Deploy auditing, alerting, TDE and SQL vulnerability scanning to SQL Databases when it does not exist in the deployment |
```

Reply:
I've just fixed up the grammar in the description, otherwise great first contribution @cassiekays